testing/test.go

```diff
@@ -33,6 +33,31 @@ func NewTestLogger(t *testing.T) logr.Logger {
 	return logr.New(l)
 }
 
+// Options carries parameters which influence the way logs are generated.
+type Options struct {
+	// LogTimestamps tells the logger to add a "ts" key to log
```

Comment:
Comment and code don't match (names are different)

Reply:
also broken in funcr, if you don't mind fixing both

Reply:
I wonder whether we should have a check for this. I completely missed it.

Anyway, fixed in both places.

Reply:
I am surprised no linter catches it.

Reply:
Answer in issue #82...